app/cisc/cisc.go

```diff
@@ -0,0 +1,312 @@
+package main
+
+import (
+	"os"
+
+	"encoding/hex"
+
+	"path"
+
+	"io/ioutil"
+	"strings"
+
+	"github.com/dedis/cothority/app/lib/config"
+	"github.com/dedis/cothority/log"
+	"github.com/dedis/cothority/services/identity"
+	"gopkg.in/codegangsta/cli.v1"
+)
+
+/*
+Cisc is the Cisc Identity SkipChain to store information in a skipchain and
```

Comment:
Put these as the firsts lines of the files so it shows up nicely in godoc (IIRC ... !)